### PR TITLE
add multi pages

### PR DESCRIPTION
**Pull request by anresa**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import {BrowserRouter} from 'react-router-dom';
```
Comment (julialoeschel) [inline src/index.js:3]: I guess you dont need the browserRouter here ;)

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import SearchResults from '../components/SearchResults/SearchResults';
+import productsDatabase from '../database.js';
+
+export default function Home() {
+  const [searchTerm, setSearchTerm] = useState('');
+  function handleSearchTermChange(changedTerm) {
+    setSearchTerm(changedTerm);
+  }
+
+  const filteredProducts = productsDatabase.reduce((prev, cur) => {
```
Comment (julialoeschel) [inline src/pages/Home.js:15]: try not to use short words. I guess this is previous and current. 
Its always easier to read when you use descriptive words.

**Review by julialoeschel — COMMENTED** [review]
Hey Anna, great work there! 

I am wondering why you already implement browserRouter, if there are no functioning pages to lead to. 
For the next time i would suggest to implemnet that later for only working pages ;) 

but for now - good job!!
:octocat:

File: src/pages/Home.js (hunk 2 above)
Comment (anresa) [reply src/pages/Home.js:15]: @julialoeschel Coach Ralf wrote this, literally. 😆

**Pull request merged**